src/filterOneSelector.js

```diff
@@ -91,8 +91,25 @@ function countMultiple(data, propFunc) {
 
 function countMultipleNegated(data, propFunc) {
     const counts = countMultiple(data, propFunc);
-    for (const key in counts) {
-        counts[key] = data.length - counts[key];
+    for (const [key, value] of counts) {
+        counts.set(key, data.length - value);
     }
     return counts;
 }
+
+function countBy(array, accessor) {
```

Comment:
dead code no?

Reply:
impossible. this file has 100% coverage. I reimplemented the countBy to return a Map because lodash returns an object